### PR TITLE
Accessibility updates to Scroller

### PR DESCRIPTION
**Pull request by MinamiFunakoshiTR**
### What's in this pull request

- [ ] Bug fix
- [x] New component/feature
- [ ] Documentation update
- [ ] Other

### Description

Hides all background graphics, adds alt text option for each step, and updates docs. See this for more detail https://github.com/reuters-graphics/graphics-components/issues/13

Details of what's changed:
- Background divs now have `aria-hidden=true` in the interactive and embed components
- Updates `docProps` to pull AltText from google doc
- Visibly hidden alt text, if it exists, is added after the foreground text in the interactive and embed components
- Updates docs to include the altText prop. Explains that you can either add alt text to each step or to just the first step if you want to describe all graphics in the scroll section in one alt text.

### CODE REVIEW

File: src/components/@types/global.ts [hunk 1]
```diff
@@ -5,11 +5,11 @@ import type { ComponentType } from 'svelte';
  */
 export type ContainerWidth = 'narrower' | 'narrow' | 'normal' | 'wide' | 'wider' | 'widest' | 'fluid';
 
-
 /**
  * A step in the Scroller component.
  */
 export interface ScrollerStep {
+  altText: string;
```
Comment (hobbes7878) [inline src/components/@types/global.ts:12]: This should be an optional prop I think, b/c you can have steps without altText.

`altText?: string;`

File: src/components/Scroller/Embedded/Foreground.svelte [hunk 2]
```diff
@@ -11,11 +11,19 @@
 {#if step.foreground === '' || !step.foreground}
   <!-- Empty foreground -->
   <div class="empty-step-foreground step-{index + 1}"></div>
+
+  {#if typeof step.altText === 'string'}
+    <p class="background-alt-text visually-hidden">{step.altText}</p>
```
Comment (hobbes7878) [inline src/components/Scroller/Embedded/Foreground.svelte:16]: Is there a case where you'd want multiple paragraph breaks in a single piece of altText? I think we talked about this already on some other component and I can't remember if we decided that wasn't a use case. 

But if it is, then can use marked to parse the content and wrap the lot in a containing div that visually hidden.

File: src/components/Scroller/docProps.ts [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import type { ComponentType } from 'svelte';
 
 interface BlockStep {
+  AltText: string;
```
Comment (hobbes7878) [inline src/components/Scroller/docProps.ts:4]: Same here. Probably should be optional: `AltText?: string;`

File: src/components/Video/docs.svx [hunk 4]
```diff
@@ -35,7 +35,7 @@ slug: video
   <Video
     ariaDescription="Description of your video for screen readers."
     src={`${assets}/videos/Sequence silent video_4.mp4`}
-    size="wide"
+    width="wide"
```
Comment (hobbes7878) [inline src/components/Video/docs.svx:38]: Looks like this is a holdover from your other PR. Make sure to merge that one in first.

**Review by hobbes7878 — CHANGES_REQUESTED** [review]
Couple changes to types and a couple questions, and I think this is good to go.

File: src/components/@types/global.ts (hunk 1 above)
Comment (MinamiFunakoshiTR) [reply src/components/@types/global.ts:12]: You're right, thanks for catching

File: src/components/Scroller/Embedded/Foreground.svelte (hunk 2 above)
Comment (MinamiFunakoshiTR) [reply src/components/Scroller/Embedded/Foreground.svelte:16]: It wouldn't hurt to add the option. I'll wrap it in marked.parse

File: src/components/Scroller/docProps.ts (hunk 3 above)
Comment (MinamiFunakoshiTR) [reply src/components/Scroller/docProps.ts:4]: done

File: src/components/Video/docs.svx (hunk 4 above)
Comment (MinamiFunakoshiTR) [reply src/components/Video/docs.svx:38]: yup will do

**Comment by MinamiFunakoshiTR** [comment]
@hobbes7878 made the fixes you suggested. Ready for your review again

**Review by hobbes7878 — APPROVED** [review]

**Pull request merged**